<ui version="4.0" >
 <class>LangPropPageForm</class>
 <widget class="QWidget" name="LangPropPageForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>334</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>-</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="GroupBox2" >
     <property name="title" >
      <string>Language Elements</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="unnamed" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Language code (ISO 639):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="l_langcode" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QGroupBox" name="GroupBox3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Articles</string>
        </property>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>11</number>
         </property>
         <property name="topMargin" >
          <number>11</number>
         </property>
         <property name="rightMargin" >
          <number>11</number>
         </property>
         <property name="bottomMargin" >
          <number>11</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <widget class="QLabel" name="male_label" >
           <property name="text" >
            <string>&amp;Male:</string>
           </property>
           <property name="buddy" >
            <cstring>def_male</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="female_label" >
           <property name="text" >
            <string>&amp;Female:</string>
           </property>
           <property name="buddy" >
            <cstring>def_female</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QLineEdit" name="indef_female" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLabel" name="TextLabel3" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Definite</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="def_female" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLineEdit" name="indef_male" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="def_male" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="def_natural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="2" >
          <widget class="QLineEdit" name="indef_natural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="natural_label" >
           <property name="text" >
            <string>&amp;Neutral:</string>
           </property>
           <property name="buddy" >
            <cstring>def_natural</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QLabel" name="TextLabel4" >
           <property name="text" >
            <string>Indefinite</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QGroupBox" name="GroupBox4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Conjugation</string>
        </property>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>11</number>
         </property>
         <property name="topMargin" >
          <number>11</number>
         </property>
         <property name="rightMargin" >
          <number>11</number>
         </property>
         <property name="bottomMargin" >
          <number>11</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <widget class="QLabel" name="pers1_label" >
           <property name="text" >
            <string>&amp;1. Person:</string>
           </property>
           <property name="buddy" >
            <cstring>first_singular</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="pers2_label" >
           <property name="text" >
            <string>&amp;2. Person:</string>
           </property>
           <property name="buddy" >
            <cstring>second_singular</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="pers3_label" >
           <property name="text" >
            <string>3. Person:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" >
          <widget class="QLineEdit" name="thirdM_plural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="6" column="2" >
          <widget class="QLineEdit" name="thirdN_plural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="2" >
          <widget class="QLineEdit" name="thirdF_plural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="2" >
          <widget class="QCheckBox" name="thirdP_common" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>C&amp;ommon</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="male_c_label" >
           <property name="text" >
            <string>M&amp;ale:</string>
           </property>
           <property name="buddy" >
            <cstring>thirdM_singular</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0" >
          <widget class="QLabel" name="female_c_label" >
           <property name="text" >
            <string>F&amp;emale:</string>
           </property>
           <property name="buddy" >
            <cstring>thirdF_singular</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="0" >
          <widget class="QLabel" name="natural_c_label" >
           <property name="text" >
            <string>Neu&amp;tral:</string>
           </property>
           <property name="buddy" >
            <cstring>thirdN_singular</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QLabel" name="TextLabel4_2" >
           <property name="text" >
            <string>Plural</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QCheckBox" name="thirdS_common" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&amp;Common</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLabel" name="TextLabel4_2_2" >
           <property name="text" >
            <string>Singular</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="second_singular" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QLineEdit" name="thirdM_singular" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <widget class="QLineEdit" name="thirdF_singular" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="6" column="1" >
          <widget class="QLineEdit" name="thirdN_singular" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="first_singular" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QLineEdit" name="first_plural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QLineEdit" name="second_plural" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>def_male</tabstop>
  <tabstop>def_female</tabstop>
  <tabstop>def_natural</tabstop>
  <tabstop>indef_male</tabstop>
  <tabstop>indef_female</tabstop>
  <tabstop>indef_natural</tabstop>
  <tabstop>first_singular</tabstop>
  <tabstop>second_singular</tabstop>
  <tabstop>thirdS_common</tabstop>
  <tabstop>thirdM_singular</tabstop>
  <tabstop>thirdF_singular</tabstop>
  <tabstop>thirdN_singular</tabstop>
  <tabstop>first_plural</tabstop>
  <tabstop>second_plural</tabstop>
  <tabstop>thirdP_common</tabstop>
  <tabstop>thirdM_plural</tabstop>
  <tabstop>thirdF_plural</tabstop>
  <tabstop>thirdN_plural</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
